WMS ID: 11408

KellyDSmith-orcl opened the pull request:
# WMS ID: 11408

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved


# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [X] Filenames are descriptive
- [X] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [X] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [X] Image references in markdown contain an alternate text

ramumg reviewed (changes requested):
@KellyDSmith-orcl, 

- Make sure your text and images are aligned correctly. Config tenancy looks like it has different alignments. 
- For notes please use this syntax: 
![image](https://github.com/oracle-livelabs/database-maa/assets/84869754/9ec46a84-9601-4cc1-8f23-25af9bb63247)
- Make sure your file names are correct
![image](https://github.com/oracle-livelabs/database-maa/assets/84869754/2c7bd176-1ec7-44ca-8750-8e69f51fb285)
- please add the need help sections to your ocw manifests before the support section. 

Thank you, 
Ramu

KellyDSmith-orcl commented:
@ramumg Adjusted text/image alignment, fixed the note format, adjusted the file name, and adjusted the help section in OCW.

anooshapilli commented:
@KellyDSmith-orcl - Please resolve the conflicting files issue and make these changes to the workshop
- In Intro lab, 
    - rename Estimated Time to Estimated Workshop Time
    - Move the Appendix section before objectives and rename it as heading 3 ### About Zero Data Loss Autonomous Recovery Service
- In the tenancy version, point the Get Started in manifest.json to this link - https://oracle-livelabs.github.io/common/labs/cloud-login/cloud-login.md
-  In the individual labs, the lab title and the title in the manifest.json file should match. Update the lab titles in markdown files to match the titles in the manifest.json file
- Indent the instruction/screenshot in Lab 1 Task 3 Step 1, Lab 3 Task 1 Step 5, Lab 4 Task 2 Step 1, 3
- In Lab 3, 4 - The link to image ham_baseDB.png is broken.

KellyDSmith-orcl commented:
@anooshapilli  I updated the estimated time label, moved the appendix, adjusted the link for the tenancy manifest, adjusted the indents, and fixed the broken image in Lab 3 & 4.   I can't find a mismatched in the title between the lab and the manifest.  Maybe I am looking at the wrong value?  Can you let me know what I should be looking at in those titles?  It won't let me resolve the conflicts.  I see a message that says "Only those with write access to this repository can merge pull requests."

anooshapilli commented:
@KellyDSmith-orcl - Please follow the steps in Lab 3 Troubleshooting section Issue 3 to resolve the conflicting files issue - https://oracle-livelabs.github.io/common/sample-livelabs-templates/create-labs/labs/workshops/livelabs/?lab=3-labs-sync-github#Troubleshooting

KellyDSmith-orcl commented:
@anooshapilli - Thanks for that pointer to Lab 3.  I think I got it worked out.  @ramumg - your change request is still open, were there any other changes?

anooshapilli commented:
@KellyDSmith-orcl - Please make this change to approve the PR - In the individual labs, the lab title and the title in the manifest.json file should match. Update the lab titles in markdown files (all the labs have same title) to match the titles in the manifest.json file.

KellyDSmith-orcl commented:
@anooshapilli - I think I figured out what you meant about the titles.  Made the updates to each md file.

KellyDSmith-orcl commented:
@ramumg - Adjusted the help section and OCW
